src/components/ProductVersionMilestonesPage/ProductVersionMilestonesPage.tsx

```diff
@@ -57,7 +57,7 @@ export const ProductVersionMilestonesPage = ({ componentId = 'm1' }: IProductVer
           {latestProductMilestone && (
             <TextContent>
               <br />
-              <Text>
```

Comment:
I have to change it to div as `Text` will introduce <p> and no <div> allowed for its children. But PF Tooltip of ProductMilestoneReleaseLabel will anyway introduce div so have no other choices.